replications/tracked/tracked_replications.go

```diff
@@ -0,0 +1,10 @@
+package tracked
```

Comment:
Defining this in its own package was the result of running into cyclic import errors when this struct was defined in queue_management.go, service.go, or its own file within the internal package. Open to suggestions if this doesn't seem like a clean solution.